ManageIQ/manageiq
# [DARGA] [EUWE] Remove disabling of 'instance_retire' button

Leaves the button "Set Retirement Date" enabled even for a retired instance

Links
----------------
https://bugzilla.redhat.com/show_bug.cgi?id=1306471

Steps for Testing/QA
-------------------------------
1. retire an instance
2. go to that instance details page
3. click `Lifecycle`
4. button `Set Retirement Date` should be enabled
 
**(see BZ ticket for more info)**
